### ISSUE TITLE
Small request: please add for each generated json a `_type` property

### ISSUE THREAD

**Issue by schittli** [yt-dlp/yt-dlp#2447]
### Checklist

- [X] I'm reporting a feature request
- [X] I've verified that I'm running yt-dlp version **2022.01.21**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

Good evening

it is great to be able to process the yt-dlp json files automatically, and then it is very useful to be able to easily detect which json file type you are working with 😃 

### For example, the downloaded playlist json file has a `_type` field:
```
{
  "_type": "playlist",
  "uploader": "…",
  "uploader_id": "…",
  "uploader_url": "…",
  "thumbnails": [
    …
```

### Request: Could you please add a `_type` property for each generated json file?
This would make a lot easier 😃 

Thanks a lot, kind regards,
Thomas

**Comment by pukkandan** [comment]
If no `_type` is present, it is to be considered as `video`

**Comment by schittli** [comment]
Hello @pukkandan 
Thanks for your answer, which unfortunately is not true: I found at least three different json types that yt-dlp generates:

1. The pure Playlist json file
2. The json file for one Video 
  for example, it does have a `playlist` field
  and a `original_url` field, too
3. The json file for a playlist. 
  for example, it does have a `playlist` field, too,
  but no `language` field, for example.

You see, it's not that simple.

But anyway: 
adding a `_type` field to all json-file generators would be very little effort, could probably be programmed without risk, and would make automatic post-processing much easier.

Therefore it is a pity, this suggestion was rejected.

**Comment by pukkandan** [comment]
What!? The issue was closed in 6a5a30f9e2f646aa48ce1ce04e54ac12d13ac7fb which did add `_type` to all info jsons.

> which unfortunately is not true: I found at least three different json types that yt-dlp generates:

The only types of infojson yt-dlp writes are `video`, `playlist` and `multi_video`. Everything except `video` already have it's `_type` written even before this commit. I have no idea what you are talking about with "the different types"
